packages/stockflux-watchlist/src/components/watchlist/Watchlist.js

```diff
@@ -24,6 +24,7 @@ const Watchlist = () => {
   const [dragOverIndex, setDragOverIndex] = useState(null);
   const [unwatchedSymbol, setUnwatchedSymbol] = useState(null);
   const [displayPreview, setDisplayPreview] = useState(false);
+  const [displayDelete, setDisplayDelete] = useState(false);
```

Comment:
Would this be any simpler if it was collapsed into one variable? Maybe `displayWindow` and then `setDisplayMode`?
So there could be three states - false to display nothing, a constant for chart state and a constant for delete state?

Reply:
That's a good idea. refactored code to have it use a single variable now,